testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/example/cors/CorsExampleAdapterTest.java

```diff
@@ -190,7 +184,6 @@ private String getAuthServerVersion() {
                 "/auth/admin/master/console/#/server-info");
         jsDriverTestRealmLoginPage.form().login("admin", "admin");
 
-        WaitUtils.waitUntilElement(By.tagName("body")).is().visible();
```

Comment:
I removed it because the `login` method is using `clickLink()` function to log in, so any wait after this should not be necessary. But I can revert it if you want.

Reply:
Thanks for the explanation. No need to revert. ;)